src/components/Swap/Input.tsx

```diff
@@ -28,22 +29,20 @@ const Balance = styled(ThemedText.Body2)`
   transition: color 0.25s ease-in-out;
 `
 
-const InputColumn = styled(Column)<{ disableHover?: boolean }>`
+const InputColumn = styled(Column)<{ disableHover?: boolean; widgetWidthPx: number }>`
```

Comment:
Instead of passing the `widgetWidth` directly to `InputColumn` and including more logic in the CSS, can you move the width check (`widgetWidth > 420`) to the `FieldWrapper` function?

Even better, if this can be labeled more abstractly (like `useIsWideWidget`) and the actual width is less important, can you move the logic into a hook within `hooks/useWidgetWidth`?